media/js/app/assetmgr/collectionwidget.js

```diff
@@ -256,9 +256,9 @@ CollectionWidget.prototype.mapEvents = function() {
                 success: function(json, textStatus, xhr) {
                     self.signalInsert(assetId, json.annotation.id);
                     self.$modal.modal('hide');
-                    return;
                 }
             });
+            return;
```

Comment:
The return at Ln. 261 is needed to prevent the code from falling into the next bit of logic and creating a global annotation when one already exists. Noting though - the return at Ln 276 is unneeded and can be removed. I'll do a PR to fix that. Thanks for the close review.